scout/server/blueprints/cases/controllers.py

```diff
@@ -663,6 +679,9 @@ def _append_evaluated_variant_by_type(
     """
     for eval_category, variant_key in CASE_REPORT_VARIANT_TYPES.items():
         if variant_key in var_obj and var_obj[variant_key] is not None:
+
+            add_bayesian_acmg_classification(var_obj)
```

Comment:
This are the other types of variants, like commented, classified, etc

Reply:
I've realised that ACMG classif is shown anyway for all classified variants, that's why this function is always called now

Reply:
Yep! 👍